### ISSUE TITLE
NullPointerException for Batch layer in pure scala when pastData == null

### ISSUE THREAD

**Issue by mzorro** [OryxProject/oryx#324]
```
16/11/08 16:45:00 ERROR scheduler.JobScheduler: Error running job streaming job 1478594700000 ms.0
java.lang.NullPointerException
        at com.cloudera.oryx.lambda.batch.ScalaBatchLayerUpdateAdapter.runUpdate(ScalaBatchLayerUpdateAdapter.java:53)
        at com.cloudera.oryx.lambda.batch.BatchUpdateFunction.call(BatchUpdateFunction.java:145)
        at com.cloudera.oryx.lambda.batch.BatchUpdateFunction.call(BatchUpdateFunction.java:50)
        at org.apache.spark.streaming.api.java.JavaDStreamLike$$anonfun$foreachRDD$2.apply(JavaDStreamLike.scala:280)
        at org.apache.spark.streaming.api.java.JavaDStreamLike$$anonfun$foreachRDD$2.apply(JavaDStreamLike.scala:280)
......
```

**Comment by mzorro** [comment]
Fix with 78848676aa9c39086a837ae8540c652b26a0a303

**Comment by srowen** [comment]
Resolved by https://github.com/OryxProject/oryx/pull/325
